File: core-java-practice/gcr-codebase/programming-elements/KilometeresToMiles.java
package Dayone;

import java.util.Scanner;

public class KilometeresToMiles {
  //write a prgrm that inputs the kilometers of distance and convert it to miles 
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
        double km = sc.nextDouble();
        double miles = km * 0.621371;
        System.out.println("Distance in miles :" + miles);
  }
}

File: core-java-practice/gcr-codebase/programming-elements/KilometeresToMiles.cs
using Systems;
namespace programming-elements{
  class KilometeresToMiles {
    //write a prgrm that inputs the kilometers of distance and convert it to miles 
    static void Main(String[] args) {
      double km = double.Parse(Console.ReadLine());
      double miles = km * 0.621371;
      Console.WriteLine("Distance in miles :" + miles);
    }
  }
}
